striatum/bandit/bandit.py

```diff
@@ -4,43 +4,60 @@
 from abc import abstractmethod
 
 
-# TODO: think about how to use this
 class Action(object):
-    """The action object"""
-    @abstractmethod
-    def __init__(self):
-        pass
+    """The action object
+
+        Parameters
+        ----------
+        action_id: int
+            The index of this action.
+
+        title: string
+            The title of this action.
+
+        content: object
+            The content of this action.
+    """
+    def __init__(self, action_id, title, content):
+        self.action_id = action_id
+        self.title = title
+        self.content = content
```

Comment:
How about retain `self.action_id`? It's the most important part of an `Action` object for contextual bandit, and adding this can avoid users' forgetting giving `action_id`.